Makefile

```
@@ -0,0 +1,5 @@
+.ONESHELL:
+
+.PHONY:ttrack-db
+ttrack-db:
```

is it used just for development right? if yes, better to have a name that forces this idea:
```suggestion
dev-dump-ttrack-db:
```